lib/tst_test.c

```diff
@@ -791,7 +791,12 @@ static void do_setup(int argc, char *argv[])
 
 	if (tst_test->mount_device) {
 		tst_test->needs_device = 1;
-		tst_test->format_device = 1;
+               // keep format device is disabled
+               // system() call is not supported. Hence, cannot format to filesystem
+               // . The block device is preformatted to ext4 at compilation
+               // and shared the details with enclave as an environment variable.
+	       // TODO: Enable below code after fixing Github issue https://github.com/lsds/sgx-lkl/issues/598
```

Comment:
This looks like spaces rather than tab is being used for indentation. It appears to be off.

Reply:
done